lib/fast_jsonapi/object_serializer.rb

```diff
@@ -292,6 +294,38 @@ def create_relationship(base_key, relationship_type, options, block)
         )
       end
 
+      ##
+      # Add an extra layer for attribute filtering to the serializer operating on the full list of defined attributes,
+      # but before any fieldset is applied. This is a more performant option than providing conditionals to every attribute.
+      #
+      # @param filter_method_name [Symbol, nil]
+      #     The name of a class method used to filter the set of attributes. This method will receive the superset of attributes,
+      #     the current record getting serialized and the serializer parameters passed along.
+      #
+      # @yieldparam filter_block [#call]
```

Comment:
This should just be a `@param`. Yieldparam would be used like:

```
@yieldparam attributes_superset
@yieldparam record
@yieldparam params
```

But since it's not actually called here, I don't think it's actually the right tag to use.